section/algorithms.html

```diff
@@ -119,6 +119,50 @@ <h3>Outgoing payment and quote grant request</h3>
             <li>
                 If |authServer| [=url/scheme=] is not `https`, return failure.
             </li>
+            <li>
+                <p>Construct a {{ GrantRequest }} |grantRequest| as follows:</p>
+                <ol>
+                    <li>
+                        Set |grantRequest|'s {{GrantRequest/client}} to <var>walletAddressDetails</var>'s {{WalletAddressDetails/url}} property.
```

Comment:
`id` is canonical. `url` can have redirects (by user or even wallet - the rest of the `WalletAddressDetails` are more likely change by `url` than by `id`) and the `jkwk.json` isn't guaranteed to be existing with `url`.

```suggestion
                        Set |grantRequest|'s {{GrantRequest/client}} to <var>walletAddressDetails</var>'s {{WalletAddressDetails/id}} property.
```